packages/react-router-native/jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Moved this config over from package.json for consistency with the other packages 👌

Reply:
👍